test/src/test/testtool/MessageStoreSenderAndListener/scenario03/out-queue.xml

```diff
@@ -15,7 +15,7 @@
   <rowset>
     <row number="0">
       <field name="MESSAGEKEY">768</field>
-      <field name="TYPE">M</field>
+      <field name="TYPE">A</field>
```

Comment:
Je hebt een nieuwe test toegevoegd en herbruiken een ander scenario's resultaat. wat voorheen een messageStore was is nu een receivers-messagelog. Hoe kan dit?